services/apps/src/apps/service/apps/de/metadata.clj

```diff
@@ -95,7 +95,7 @@
       (throw+ {:type  :clojure-commons.exception/bad-request-field
                :error (str "Rating must be an integer between 1 and 5 inclusive."
                                 " Invalid rating (" rating ") for App ID " app-id)}))
-    (when-not (:is_public app)
+    (when-not (contains? (perms-client/get-public-app-ids) app-id)
       (throw+ {:type  :clojure-commons.exception/bad-request-field
                :error (str "Unable to rate private app, " app-id)}))
```

Comment:
Now that app rating isn't tied to confluence wiki pages for public apps, I wonder if it's time to revisit this policy?